src/DB_Command.php

```diff
@@ -45,18 +45,19 @@ class DB_Command extends WP_CLI_Command {
 	 * A list of incompatible SQL modes.
 	 *
 	 * Copied over from WordPress Core code.
+	 *
 	 * @see https://github.com/WordPress/wordpress-develop/blob/5.4.0/src/wp-includes/wp-db.php#L559-L572
 	 *
 	 * @var string[] Array of SQL mode names that are incompatible with WordPress.
 	 */
-	protected $sql_incompatible_modes = [
+	protected $sql_incompatible_modes = array(
```

Comment:
Can you please revert changes in this and similar unrelated lines? For fixing code sniffer issues, please use `composer run phpcbf`. It seems you are using separate sniffer rules here.